h5p-bildetema/src/utils/language.utils.ts

```diff
@@ -7,3 +8,18 @@ export const translatedLabel = (
 ): string => {
   return translations[`lang_${language.code}`];
 };
+
+/**
+ * Filter out languages that are not in the database.
+ * This is to avoid errors when the database is updated.
+ */
+export const filterFavoriteLanguages = (
+  favoriteLanguages: Language[],
+  languagesFromDB: Language[] | undefined,
+): Language[] => {
+  return favoriteLanguages
+    .filter(language => languagesOriginal?.[language.code])
+    .filter(
+      language => languagesFromDB?.find(lang => lang.code === language.code),
+    );
```

Comment:
Not very important, but it would be nice to combine these such that we don't iterate twice.

Reply:
yess